python/code_to_gast/py_assign.py

```diff
@@ -39,7 +39,7 @@ def augop_to_str(op):
         ast.Add: "+=", 
         ast.Mult: "*=", 
         ast.Div: "/=", 
-        ast.Sub: "-=",
```

Comment:
I think the google style guide will add this trailing comma back. At least that's what it does in JS :)